modules/nf-core/pangolin/tests/main.nf.test

```diff
@@ -0,0 +1,58 @@
+nextflow_process {
+
+    name "Test Process PANGOLIN"
+    script "../main.nf"
+    process "PANGOLIN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "pangolin"
+
+    test("sarscov2 genome [fasta]") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                            [ id:'test' ], // meta map
+                            [ file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 genome [fasta] - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = [
+                            [ id:'test' ], // meta map
+                            [ file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true) ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.report.collect { file(it[1]).getName() } +
+                                process.out.versions).match() }
```

Comment:
Empty hash in stube tests shouldn't cause file to fail nf-tests anymore! https://github.com/nf-core/tools/pull/2571

Reply:
will update this